plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectWizard.java

```diff
@@ -38,17 +38,18 @@
 import org.eclipse.ui.IWorkbench;
 import org.eclipse.ui.ide.undo.WorkspaceUndoUtil;
 
-public abstract class AppEngineProjectWizard extends Wizard implements INewWizard {
+public abstract class AppEngineProjectWizard<T extends AppEngineWizardPage> extends Wizard
```

Comment:
Ah, sorry. It is the page classes, but still, I wonder why we need parameterization. Maybe I am missing something?